plugins/parsers/statsd/parser.go

```diff
@@ -0,0 +1,272 @@
+package statsd
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"io"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/metric"
+	"github.com/influxdata/telegraf/plugins/parsers/graphite"
+)
+
+const (
+	defaultFieldName = "value"
+	defaultSeparator = "_"
+	defaultTagKey    = "statsd_type"
```

Comment:
I think we should use `metric_type` to remain more compatible with stats parsed from the statsd input, along with the long names: `counter`, `gauge`, etc.

Reply:
Done that.